benchmark/utils.go

```diff
@@ -40,18 +40,15 @@ func GetImageListFromCsv(csvLoc string) ([]ImageDescriptor, error) {
 	}
 	var images []ImageDescriptor
 	for _, image := range csv {
-		if len(image) < 3 {
+		if len(image) < 4 {
 			return nil, errors.New("image input is not sufficient")
 		}
-		var sociIndexManifestRef string
```

Comment:
This change was made specifically for stargz test which does _not_ have soci index

Reply:
Seems like I was using an outdated test image set. Apologies and feel free to close.